github/resource_github_team_sync_group_mapping_test.go

```diff
@@ -0,0 +1,97 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-github/v29/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccGithubTeamSyncGroupMapping_basic(t *testing.T) {
+	var team github.Team
+
+	rn := "github_team.foo"
+	randStringTeam := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	teamName := fmt.Sprintf("tf-acc-test-%s", randStringTeam)
+	randStringGroup := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	groupID := acctest.RandString(5)
+	groupName := fmt.Sprintf("tf-acc-test-group-%s", randStringGroup)
+	description := "Terraform acc test group"
+	updatedDescription := "Terraform acc test group - updated"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGithubTeamSyncGroupMappingDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGithubTeamSyncGroupMappingConfig(teamName, groupID, groupName, description),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
You shouldn't need the `Check` here, if you add a step after this (similar to your last step) that will import, because the import will check the state before and the state after, basically doing the same thing.

Reply:
will these checks for specific attributes help do the checks instead? rather than check if the actual resource exists